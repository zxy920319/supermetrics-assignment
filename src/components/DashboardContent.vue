<template>
  <div class="min-h-screen bg-gray-200 pb-20">
    <div class="bg-[#03A9F4] px-3 md:px-8 h-56" />

    <div class="px-20 -mt-24">
      <div class="container mx-auto max-w-full">
        <slot></slot>
      </div>
    </div>
  </div>
</template>
