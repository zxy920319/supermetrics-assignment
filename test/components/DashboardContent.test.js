import DashboardContent from "@/components/DashboardContent.vue"
import { shallowMount } from "@vue/test-utils"
import { describe, expect, it } from "vitest"

describe("DashboardContent", () => {
  it("renders component", () => {
    const wrapper = shallowMount(DashboardContent)

    expect(wrapper.html()).toMatchSnapshot()
  })
})
