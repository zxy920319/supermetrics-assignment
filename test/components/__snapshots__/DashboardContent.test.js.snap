// Vitest Snapshot v1

exports[`DashboardContent > renders component 1`] = `
"<div class=\\"min-h-screen bg-gray-200 pb-20\\">
  <div class=\\"bg-[#03A9F4] px-3 md:px-8 h-56\\"></div>
  <div class=\\"px-20 -mt-24\\">
    <div class=\\"container mx-auto max-w-full\\"></div>
  </div>
</div>"
`;
